e2e/app.e2e.spec.ts

```diff
@@ -8,11 +8,13 @@ import { appConfig } from '../src/common/test.utils';
 import { authTestSuite } from './auth';
 import { claimTestSuite } from './claim';
 import { statusList2021TestSuite } from './status-list';
-import { shutDownIpfsDaemon, spawnIpfsDaemon } from './setup-ipfs';
+import { shutdownIpfsCluster, spawnIpfsCluster } from './setupIpfsCluster';
```

Comment:
@JGiter We are not bringing up `IPFSGateway` for test cases, just relying on `Cluster` tests?